certbot/client.py

```diff
@@ -240,6 +241,9 @@ def __init__(self, config, account_, auth, installer, acme=None):
                 auth, self.acme, self.account, self.config.pref_challs)
         else:
             self.auth_handler = None
+        if sys.version_info[:2] == (3, 3):
+            logger.warn("Python 3.3 support will be dropped in the next release"
+                        "of Certbot. Please upgrade your Python version")
```

Comment:
Tiny niggle: the \_\_init__.py warning ends in a fullstop.

See also http://www.python3statement.org/practicalities/ for some ideas.